crypt-common.h

```diff
@@ -30,4 +30,24 @@ const unsigned char ascii64[65];
 #define b64t   ((const char *) ascii64)
 #define itoa64 ascii64
 
+/* For historical reasons, crypt and crypt_r are not expected ever to
+   return 0, and for internal implementation reasons (see
+   call_crypt_fn, below), it is simpler if the individual algorithms'
```

Comment:
Cut-and-paste error here: the parenthetical should read "see call_crypt_fn, in crypt.c".

Reply:
Fixed.